integreat_compass/cms/views/interactions/vote_form_view.py

```diff
@@ -0,0 +1,87 @@
+from django.db.models import Case, Count, When
+from django.shortcuts import render
+from django.views.generic import TemplateView
+
+from ...constants import offer_version_states
+from ...models import OfferVersion, User
+
+
+class VoteFormView(TemplateView):
+    """
+    View for the vote form
+    """
+
+    template_name = "interactions/vote_form.html"
+
+    @staticmethod
+    def get_pending_offer_versions(request, number_of_board_members):
+        """
+        Get all offer versions that need a vote from currently logged-in user
+
+        :param request: Object representing the user call
+        :type request: ~django.http.HttpRequest
+
+        :param number_of_board_members: current number of board member
+        :type number_of_board_members: int
+
+        :return: list of offer versions with all details
+        """
+        latest_version_of_offer = OfferVersion.objects.order_by(
+            "offer", "-created_at"
+        ).distinct("offer")
+        offer_versions = (
+            OfferVersion.objects.prefetch_related("votes")
+            .select_related("offer")
+            .select_related("offer__organization")
+            .prefetch_related("offer__tags")
+            .annotate(approves=Count(Case(When(votes__approval=True, then=1))))
+            .annotate(declines=Count(Case(When(votes__approval=False, then=1))))
+            .annotate(pending=number_of_board_members - Count("votes"))
+            .annotate(user_voted_on=Case(When(votes__creator=request.user, then=True)))
+            .order_by("-user_voted_on")
+            .filter(pk__in=latest_version_of_offer)
+        )
+        pending_offer_versions = list(
+            filter(
+                lambda offer_version: offer_version.state
+                == offer_version_states.PENDING,
+                offer_versions,
+            )
+        )
+        for version in pending_offer_versions:
+            version.user_vote = version.votes.filter(creator=request.user).first()
+        return pending_offer_versions
+
+    def get(self, request, *args, **kwargs):
+        r"""
+        Render :class:`~integreat_compass.cms.forms.interactions.vote_form.VoteForm`,
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        :param \*args: The supplied arguments
+        :type \*args: list
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :raises ~django.core.exceptions.PermissionDenied: If user does not have the permission to edit the specific page
+
+        :return: The rendered template response
+        :rtype: ~django.template.response.TemplateResponse
+        """
+        number_of_board_members = User.objects.filter(
+            groups__name="BOARD_MEMBER"
+        ).count()
```

Comment:
```suggestion
        number_of_board_members = max(User.objects.filter(
            groups__name="BOARD_MEMBER"
        ).count(), settings.NEW_OFFER_GREMIUM_SIZE
```
Requires settings import.